### ISSUE TITLE
[FR] Fade animation and ken burns effect

### ISSUE THREAD

**Issue by TheZoker** [giejay/Immich-Android-TV#72]
**Is your feature request related to a problem? Please describe.**
What I like about the screensaver from google tv or fire tv, is the way the transition is happening. 

**Describe the solution you'd like**

It would be nice to replicated this look with these options:
- Add ken burns effect ([demo](https://demo.wocode.com/fx-ken-burns))
- Allow to change transition from slide to fade
- Allow to set animation duration (like 2 seconds etc.)

**Additional context**

**Comment by giejay** [comment]
I will add the possbility to set one of the transformers mentioned here: https://github.com/geftimov/android-viewpager-transformers?tab=readme-ov-file

Hope that will suffice.

**Comment by TheZoker** [comment]
Not really since none of those is close to a smooth fade in-out transition. There is even an open issue for it:
https://github.com/geftimov/android-viewpager-transformers/issues/7

The current implementation (and also all of the demos in your link) are just to hectic and not like smooth and subtle as at least I would want it to be. In my opinion the Fire TV slideshow is a very good implementation of a nice background screensaver, which looks nice and is not to hectic

**Comment by giejay** [comment]
If you can find the source code for that, I will give it a try. Otherwise, it will not be a high priority for me.

**Comment by TheZoker** [comment]
Is maybe one of those sufficient?

- https://stackoverflow.com/questions/25096069/viewpager-animation-fade-in-out-instead-of-slide
- https://gist.github.com/RealAndyLawton/7606451

**Comment by giejay** [comment]
Thanks, I will try this somewhere this week

**Comment by giejay** [comment]
The pageTransformers do not work with my current approach of forwarding items in the ViewPager, it's a known bug at Google (https://github.com/android/views-widgets-samples/issues/109). For now, I used a different approach by using a scroller and a configurable slide animation time. This is not the same as fade/in/out but I did not find a solution for that unfortunately. Maybe in a new release of the viewpager, this will get fixed, for now, I will close this.
